### ISSUE TITLE
Docker-Container not booting up

### ISSUE THREAD

**Issue by ahoiroman** [mikecao/umami#1232]
In the latest release (v. 1.33) we got this issue:

> $ npm-run-all check-db start-server
/bin/sh: npm-run-all: not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by Olakeen** [comment]
Same issue on Railway

**Comment by rogerclotet** [comment]
Same here :)
It seems like the container was not built properly and dependencies were not installed.

**Comment by meepmeep** [comment]
(same issue with umami:mysql-latest)

**Comment by ariaieboy** [comment]
using latest postgresql image and i get the same error

**Comment by mikecao** [comment]
Thanks for the report, we'll get a new image out.

**Comment by mikecao** [comment]
New images are out and should be working now. Let me know if you run into any more issues.

**Comment by ahoiroman** [comment]
Is this procedure documented for docker-environments already? That would be great :-)

**Comment by Nalsai** [comment]
I added `command: "prisma migrate resolve --applied 20210320112717_init"` to the umami (not db) service in my docker-compose.yml and ran it once to run the migration instead of running umami. Afterwards I removed it again to run umami normally again.

It should also be possible with a simple one time docker run command to run the umami container with the `prisma migrate resolve --applied 20210320112717_init` command, while the db is running.

**Comment by saschwarz** [comment]
Yeah I'm not familiar w/prisma at all. I've used other frameworks like Django where you can issue `python manage.py migrate` and it runs only the unapplied migrations (in order). So then the Dockerfile can always safely contain that command (as long as you only run a single instance of the django container).

**Comment by Nalsai** [comment]
I'm not familiar with prisma either. But I think there should be some way to have migrations be run automatically (maybe you could have it be disabled by default and only enabled manually at your own risk). The migration this time wouldn't have been a problem if it ran automatically, at least on my setup. Afterwards it seems to mark it as done, so the migration isn't run again.

**Comment by mikecao** [comment]
Migrations can be run automatically, but we opted not to just in case things go wrong. Maybe we can automatically run the script if you pass in an environment variable?

**Comment by saschwarz** [comment]
@mikecao I would greatly appreciate it if you could drop the `docker-compose` command(s) to run the migration in a comment here or added to the docs for those of us who are less experienced. This is my *guess* for the upgrade for this release:
```
docker-compose pull
docker-compose run umami prisma migrate resolve --applied 20210320112717_init
docker-compose up --force-recreate
```

**Comment by ariaieboy** [comment]
> @mikecao I would greatly appreciate it if you could drop the `docker-compose` command(s) to run the migration in a comment here or added to the docs for those of us who are less experienced. This is my _guess_ for the upgrade for this release:
> 
> ```
> docker-compose pull
> docker-compose down  # ???
> docker-compose run umami prisma migrate resolve --applied 20210320112717_init  # ???
> docker-compose up --force-recreate
> ```

if I wasn't wrong the `20210320112717_init` is dynamic and you need to run the check-db scripts to generate that. 



> Migrations can be run automatically, but we opted not to just in case things go wrong. Maybe we can automatically run the script if you pass in an environment variable?

if it's not overriding all tables with data in it and purging all the data I think we should have an option to run migrations automatically.

for example in laravel we have `PHP artisan migrate` that will check the DB with migrations and if there is any new migration since the last migrate it will only run those migration on DB

**Comment by userdehghani** [comment]
same here but i ran the following command on container boot-up by accessing `sh` before facing with error. eventually it fixed:

```
prisma migrate resolve --applied 20210320112717_init
```

**Comment by mikecao** [comment]
This should be a smoother experience in `v1.33.1`. Sorry for the confusion.

**Comment by brunoamaral** [comment]
@Nalsai thank you for sharing the fix, I was feeling lost on how to get around the container not booting 🙂

**Comment by wheeleran** [comment]
@brunoamaral the v1.33.1 that @mikecao released fixed the issue, no prisma migrate needed.

**Comment by MiguelNdeCarvalho** [comment]
Everything working on `v1.33.1`. Thanks  @mikecao!
